src/infrastructure/jira/jira-service.ts

```diff
@@ -222,10 +222,12 @@ class JiraService {
 		} catch (error) {
 			if (error instanceof NotFoundHttpClientError) {
```

Comment:
**question:** @stannnous , what do you think about always overwriting `attached-design-url` on linking a design? Currently, the app writes to this property only if it is empty.

This could help to avoid data consistency issues in some edge cases (e.g., a design was unlinked but Connect app didn't remove the design URL from `attached-design-url`). I do not know such use cases but it looks like just a more reliable solution.

The side effect of this would be the "Jira" widget in Figma always showing the **last** linked design instead of the one linked from widget or the first one linked from Jira 🤯

Reply:
What does the existing Figma app do if you add a second design in Jira? Does it update the `attached-design-url` or keep the original value?

I think your suggestion makes sense but I would err on maintaining the existing behaviour.